Android/Guava/source/Guava.FailureAccess/Guava.FailureAccess.csproj

```diff
@@ -20,38 +20,30 @@
     <Title>Google Guava FailureAccess reference library for Xamarin.Android</Title>
     <Summary>Xamarin.Android bindings for Google Guava FailureAccess</Summary>
     <Description>
-    NOTE: This package is meant to be used as a reference only for other binding projects that depend on Guava.
-    This package does not expose any useful API's in the .dll itself.
-    
-    Guava is a set of core libraries that includes new collection types (such as multimap and multiset), immutable collections, a graph library, functional types, an in-memory cache, and APIs/utilities for concurrency, I/O, hashing, primitives, reflection, string processing, and much more!
+NOTE: This package is meant to be used as a reference only for other binding projects that depend on Guava.
+This package does not expose any useful API's in the .dll itself.
+
+Guava is a set of core libraries that includes new collection types (such as multimap and multiset), immutable collections, a graph library, functional types, an in-memory cache, and APIs/utilities for concurrency, I/O, hashing, primitives, reflection, string processing, and much more!
```

Comment:
Even though indents make the .csproj look nice, it messes up the nuget.org page: 
![image](https://user-images.githubusercontent.com/1096616/63857871-bd582200-c9a4-11e9-95b3-eef834b98272.png)